src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3488,10 +3496,16 @@ static void window_ride_operating_mouseup(rct_window* w, rct_widgetindex widgetI
             set_operating_setting(
                 w->number, RideSetSetting::Departure, ride->depart_flags ^ RIDE_DEPART_WAIT_FOR_MINIMUM_LENGTH);
             break;
+        case WIDX_MINIMUM_LENGTH:
+            window_text_input_raw_open(w, WIDX_MINIMUM_LENGTH, STR_ENTER_NEW_VALUE, STR_ENTER_NEW_VALUE, _minWaitInputText, 4);
```

Comment:
Why the raw variant, specifically? The Water window uses `window_text_input_open` and it uses a simple inter input as well.